SF iOS/SF iOS/Location/FeedFetch/FeedFetchService.m

```diff
@@ -22,8 +22,8 @@ - (instancetype)init {
     return self;
 }
 
--(void)getFeedWithHandler:(FeedFetchCompletionHandler)completionHandler {
-    FeedFetchOperation *operation = [[FeedFetchOperation alloc] initWithCompletionHandler:^(NSArray<NSDictionary *> *feed, NSError *_Nullable error) {
+-(void)getFeedAtURLString:(NSString*)endpoint withHandler:(FeedFetchCompletionHandler)completionHandler {
+    FeedFetchOperation *operation = [[FeedFetchOperation alloc] initWithURLString:endpoint withCompletionHandler:^(NSArray<NSDictionary *> *feed, NSError *_Nullable error) {
```

Comment:
As far as I remember the convention for the second parameters would be to drop the `with` so `initWithURLString:completionHandler:` would be preferable.